app/controllers/ticket.controller.js

```diff
@@ -0,0 +1,26 @@
+var verification = require('../utils/verification.util');
+var Ticket = require('mongoose').model('Ticket');
+
+exports.createTicket = function(req, res) {
+    var subject = (req.body.subject) ? req.body.subject.trim() : '';
+    var message = (req.body.message) ? req.body.message.trim() : '';
+
+    if (subject === '') {
+        res.json({successful: false, text: 'Please provide a meaningful subject for the ticket.'});
```

Comment:
change
`res.json({successful: false, text: 'Please provide a meaningful subject for the ticket.'});`
to
`res.status(400).send({successful: false, text: 'Please provide a meaningful subject for the ticket.'});`

I've changed the user controller functions in this way as well in my email verification branch. This is to follow convention for http requests. As it is right now, all of the responses send a 200 status which means that the request was successful even if it wasn't.